Week3/Mohammad-Darwesh-homework-JS1-week3/mohammaddarwesh.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+//String and Array challenges
+// task 1:
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/,/g, " ");
+console.log(myString);
+// or :
+console.log(myString.replace(/,/g, " "));
+
+// task 2:
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+console.log(favoriteAnimals);
+favoriteAnimals.push("turtle");
+console.log(favoriteAnimals);
+favoriteAnimals.splice (1, 0, "meerkat");
+console.log("I think the new value of the array is [\"blowfish\", \"meerkat\", \"capricorn\", \"giraffe\", \"turtle\"]");
+console.log(favoriteAnimals);
+console.log("The array has a length of",favoriteAnimals.length);
+// or :
+const l = favoriteAnimals.length;
```

Comment:
Be careful with the names of variables, you know what `l` means right now but the next developer probably doesn't know. So it's better to give variable the correct names, `animalsLength`  might be better here.

Reply:
Hello Paul,
Thank you for your feedback .
I will use your useful advice on next homework.